### DISCUSSION TITLE
windowbuilder not working

### DISCUSSION THREAD

**Discussion by michele981gmail** [eclipse/windowbuilder#678] [Q&A]
Good morning,
I installed windowbuilder on eclipse but whe I create a Jframe i get the error in screenshot

I'm using Eclipse Version: 2022-12 (4.26.0) and JRE 17.0.9

Thanks
![Immagine](https://github.com/eclipse-windowbuilder/windowbuilder/assets/102143876/49646861-2155-44c7-97ad-5e8775049439)

**Comment by merks** [comment]
Is there any reason not to install 2023-12 (4.30) instead?  There have been an awful lot of fixes and improvements in the past year.

- https://www.eclipse.org/downloads/packages/
- https://download.eclipse.org/releases/2023-12/

**Reply by michele981gmail** [reply]
Thanks merks,
I solved using 2023-12 (4.30)
